src/Uno.UI/UI/Xaml/DependencyObjectExtensions.cs

```diff
@@ -438,12 +438,6 @@ internal static void InvalidateArrange(this DependencyObject d)
 			uielement?.InvalidateArrange();
 		}
 
-		internal static void InvalidateRender(this DependencyObject d)
-		{
-			var uielement = d as UIElement ?? d.GetParents().OfType<UIElement>().FirstOrDefault();
-			uielement?.InvalidateRender();
-		}
-
```

Comment:
👍 This method was useless as 100% self recursive,  I already had to remove it in my android's composition branch ^^